### ISSUE TITLE
Remove or archive?

### ISSUE THREAD

**Issue by icemac** [zopefoundation/zope2docs#1]
If the contents of this repository have been moved to the Zope repository does it still make sense to keep the repos and its content?

Could it be an idea to remove the content and keep the readme?

**Comment by hannosch** [comment]
Yep, that makes sense. I'd create a tag right before the removal, so it's easier to browse the history of the docs on their own.

**Comment by jamadden** [comment]
I’m not sure *all* the docs made it to the Zope repo, specifically the tal specs, but I have a potential new repo and site for those. See https://github.com/zopefoundation/zope.tal/issues/9

**Comment by hannosch** [comment]
We are only talking about the docs in this repo. Those are the Zope 2/4 specific books.

I’m not sure if there were other docs in other places in SVN and whether those got converted to git.

**Comment by jamadden** [comment]
I think we are talking about the same set of docs. The most well known (or most linked-to) copy of the TAL, TALES and METAL spec is https://docs.zope.org/zope2/zope2book/AppendixC.html corresponding to https://github.com/zopefoundation/zope2docs/blob/master/docs/zope2book/AppendixC.rst 

The last time I checked, the docs at zope.readthedocs.io (and possibly the repo at https://github.com/zopefoundation/Zope/tree/master/docs) did not have a copy of that. I just now checked again and I see that it [is published](https://zope.readthedocs.io/en/latest/zope2book/AppendixC.html) and [in the repo](https://github.com/zopefoundation/Zope/tree/master/docs/zope2book). (I was working from my phone last night.)

**Comment by icemac** [comment]
As zope.org is is still down (since more than two weeks now 😢) I started to change some links in the zopefoundation repositories which pointed to AppendixC on docs.zope.org to point to [RTD](https://zope.readthedocs.io/en/latest/zope2book/AppendixC.html). I think this new URL is currently stable and will not go away soon. Later on they can be changed to the solution discussed in zopefoundation/zope.tal#9.

**Comment by jamadden** [comment]
https://pagetemplates.readthedocs.io is up and running and the ZTK projects have been updated to point to it.

I wonder if the new [Zope 2 AppendixC](https://zope.readthedocs.io/en/latest/zope2book/AppendixC.html) could now just point to that domain.
